web/themes/custom/vote_gov/templates/partial/register/registration-type--in-person.html.twig

```diff
@@ -0,0 +1,52 @@
+  {# How to register to vote #}
+  <h2>{{ 'How to register to vote in @state_name' | t({ '@state_name': label | render }) }}</h2>
```

Comment:
I would suggest creating a variable for the state name in the reg type templates that you can replace `label | render`: `{% set state_name = label | field_value | render %}`
This makes is reusable and also fixes the space issue on the display.